var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -30,9 +30,10 @@ class Libfabric(AutotoolsPackage):
        fabric communication services to applications."""
 
     homepage = "https://libfabric.org/"
-    url      = "https://github.com/ofiwg/libfabric/releases/download/v1.5.0/libfabric-1.5.0.tar.gz"
+    url      = "https://github.com/ofiwg/libfabric/releases/download/v1.6.0/libfabric-1.6.0.tar.gz"
 
-    version('1.5.0', 'fda3e9b31ebe184f5157288d059672d6')
```

Comment:
Is there any reason to remove the 1.5.0 version? We generally keep old versions in the package unless they are no longer available for download.

Reply:
Several (~100) bug fixes for the 1.5 series have gone into 1.5.3 since the original 1.5.0 release. It would make more sense to keep the latest bugfix versions for all series, including older ones, than keeping every release in every series. The APIs did not change between 1.5.0 and 1.5.3, so there is no real reason why a user should pull in a buggier 1.5.0 when 1.5.3 is available.

Reply:
The main reason is reproducibility. If someone wants to reproduce the results from an old publication, they should be able to install the exact same libraries to do so, bugs and all. A secondary reason is for offline users who work on an air-gapped network. Spack should be able to install the packages they need from the tarballs they downloaded to a mirror. If the versions disappear, so will Spack's support for those versions.

I completely agree that we should add 1.5.3, and the latest version should always be the default. But if someone really wants to use 1.5.0, I won't stop them.

Reply:
Fair enough, I did not think of the air-gapped scenario. I'll add rework this bit then.